pkg/controller/windowsmachineconfig/windows/windows.go

```diff
@@ -59,7 +59,7 @@ func (vm *Windows) runBootstrapper() error {
 		return errors.Wrap(err, "error running bootstrapper")
 	}
 	if len(stderr) > 0 {
-		log.Info(stderr, "err", "bootstrapper initialization failed")
+		log.Info("bootstrapper initialization failed", "err", stderr)
```

Comment:
Please make this `stderr` instead of `err`, so that it is clear we are looking at stderr output.